_posts/2015-11-03-what-you-can-learn-when-you-pair-and-collaborate.md

```diff
@@ -0,0 +1,195 @@
+---
+layout: post
+name: what-you-can-learn-when-you-pair-and-collaborate
+title: 'What you can learn when you pair and collaborate'
+date: 2015-11-03 09:00:00 +00:00
+author: Mani Sarkar
+image:
+    src: /assets/img/custom/blog/why-collaborate.png
+    attribution:
+        text: None of us is as smart as all of us - Kenneth H. Blanchard
+        href: https://kostiukg.files.wordpress.com/2014/12/why-collaborate.png
+---
+
+Its been over a month since I joined 'Codurance', a formidable force, and its about time that I share my experiences with my fellow mates and the community around me.
+
+During my first few weeks at Codurance I have been busy learning various things that has been chalked out for becoming a craftsman (an Apprentice on its journey to craftsmanship).
```

Comment:
Comma between "Codurance" and "I"
craftsman and apprentice with the same first letter capitalization (both or none)